contents/tutorials/nextjs-error-monitoring.md

```diff
@@ -238,12 +238,23 @@ You can also create a similar `global-error.jsx` file to capture errors affectin
 
 Because backend requests in Next.js vary between server-side rendering, short-lived processes and more, we can't rely on exception autocapture.
 
-Instead, we create a [`instrumentation.js`](https://nextjs.org/docs/app/building-your-application/optimizing/instrumentation) file at the root of our project and set up an `onRequestError` handler there. Importantly, we to both check the request is running in the `nodejs` runtime to ensure PostHog works and get the  `distinct_id` from the cookie to connect the error to a specific user.
+The context of a user and their session lives on the frontend. As such it is necessary to pass this data to your backend when making requests. You can set PostHog up to send this data on every request by specifying the domains you wish to patch with the session and user distinct ID. 
```

Comment:
Do we recommend doing one over the other or trying one method first? I would make it very clear (when given options people for some reason just freeze lol